galsim/photon_array.py

```diff
@@ -707,3 +707,21 @@ def applyTo(self, photon_array, local_wcs=None):
         photon_array.dxdz /= factor
         photon_array.dydz /= factor
         photon_array.flux = np.where(np.isnan(factor), 0.0, photon_array.flux)
+
+
+class FocusDepth(object):
+    """Surface-layer operator that focuses/defocuses photons by changing the height of the focal
+    surface with respect to the conical beam.
+
+    Parameters:
+        depth:   The z-distance by which to displace the focal surface, in units of pixels.  A
+                 positive (negative) number here indicates an extra- (intra-) focal sensor height.
```

Comment:
This could use a translation into non-optics jargon I think.  Like "In other words, using depth>0 will cause the focus to be above the sensor.  Using depth < 0 will cause the focus to be in (or below) the surface of the sensor."  I *think* that's right anyway.